src/app/components/Footer/index.tsx

```diff
@@ -85,23 +103,46 @@ export const Footer = memo(() => {
                 <NoReleaseLink />
               ),
               CommitLink: (
-                <Anchor
-                  href={`${githubLink}commit/${process.env.REACT_APP_BUILD_SHA}`}
-                  label={process.env.REACT_APP_BUILD_SHA.substring(0, 7)}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                />
+                <span>
+                  <Anchor
+                    onClick={toggleCommitLink}
+                    label={
+                      showCommitLink
+                        ? t('footer.hideDetails', '[hide details]')
+                        : t('footer.showDetails', '[show details]')
+                    }
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  />
+                  <span>
+                    {showCommitLink && process.env.REACT_APP_BUILD_SHA && (
+                      <span>
+                        <br />
+                        (commit:
+                        <Anchor
+                          href={`${githubLink}commit/${process.env.REACT_APP_BUILD_SHA}`}
+                          label={process.env.REACT_APP_BUILD_SHA.substring(0, 7)}
+                          target="_blank"
+                          rel="noopener noreferrer"
+                        />
+                        ) built at {buildTime}
+                      </span>
+                    )}
+                  </span>
+                </span>
               ),
             }}
-            defaults="Version: <ReleaseLink/> (commit: <CommitLink/>) built at {{buildTime}}"
-            values={{
-              buildTime: intlDateTimeFormat(Number(process.env.REACT_APP_BUILD_DATETIME)),
-            }}
+            defaults="Version: <ReleaseLink/> <CommitLink/>"
           />
-          {poweredByLabel && <Box align="center">{poweredByLabel}</Box>}
         </Text>
       )}
-
+      <Text size="small" textAlign="center" margin={{ top: 'small' }}>
+        <Trans
+          i18nKey="footer.feedback"
+          t={t}
+          defaults="We’d love your feedback at wallet@oasisprotocol.org"
```

Comment:
wallet@oasisprotocol.org should be mailto: link

Reply:
Thanks, I have updated it.